lib/money.rb

```diff
@@ -41,9 +41,9 @@ class << self
     def default_currency=(val)
       block =
         if val.respond_to?(:call)
-          -> { Money::Currency.new(val.call) }
```

Comment:
Sure thing, what do you mean by the last version though?

I don't think we should drop support for inferring currency from the returned string or symbol, since it doesn't require a lot of a handling, but is a nice feature that I'm sure people are using.